src/operator/numpy/np_tensordot_int_axes_op-inl.h

```diff
@@ -0,0 +1,213 @@
+/*
```

Comment:
Why do you need a separate file for this op? If this is part of tensordot's implementation it should just fall in your `np_tensordot_op` files.

Reply:
Fixed. Thx.